src/components/App.jsx

```diff
@@ -1,43 +1,81 @@
 /* global cozy */
-import React, { useState, useEffect } from 'react'
+import React, { useState, useEffect, useCallback } from 'react'
 import { PropTypes } from 'prop-types'
 import flow from 'lodash/flow'
 
+import { useClient } from 'cozy-client'
 import { Main, Layout } from 'cozy-ui/transpiled/react/Layout'
 import Alerter from 'cozy-ui/transpiled/react/Alerter'
 import { Title } from 'cozy-ui/transpiled/react/Text'
 import { useI18n, useBreakpoints } from 'cozy-ui/transpiled/react'
 import { Sprite as IconSprite } from 'cozy-ui/transpiled/react/Icon'
 import flag, { FlagSwitcher } from 'cozy-flags'
 import 'cozy-ui/transpiled/react/stylesheet.css'
+import log from 'cozy-logger'
 
 import withContactsMutations from '../connections/allContacts'
 import ContactsSelectionBar from './layout/ContactsSelectionBar'
 import { ModalManager } from '../helpers/modalManager'
-import { initFlags } from '../helpers/flags'
+import useFlags from './Hooks/useFlags'
 import container from './AppContainer'
 import ContentWrapper from './ContentWrapper'
+import SpinnerContact from './Components/Spinner'
+import { fetchNormalizedServiceByName } from '../helpers/fetches'
 
 const ContactsApp = props => {
   // HACK to avoid CozyBar error :
   // you tried to use the CozyBar API (BarCenter) but the CozyBar is not initialised yet via cozy.bar.init
   // TODO : TO BE REMOVED
   const [cozyBarHack, setcozyBarHack] = useState(false)
 
+  const [serviceToLaunch, setServiceToLaunch] = useState(null)
+  const [hasServiceBeenLaunched, setHasServiceBeenLaunched] = useState(null)
+
+  useFlags()
```

Comment:
I personally find it a bit weird when we have custom hooks that return nothing, but… it's not wrong 😁. No need to change it imo, but i wonder if that pattern is encouraged or not.

Reply:
Oui j'ai eu la même réflexion de mon côté et j'ai hésité entre ça ou mettre un useEffect avec l'appel dedans... c'est un hook qui ne retourne rien mais je me suis dit qu'au moins il était autonome avec son useEffect embarqué.
Je peux rechanger sans problème si c'est choquant.